eng/illink.targets

```diff
@@ -52,22 +52,28 @@
     <None Include="@(ILLinkSubstitutionsXmls)" />
   </ItemGroup>
 
-  <!-- Custom binplacing for pre/post-trimming and reports that is useful for analysis
+  <!-- Custom binplacing for post-trimming and reports that is useful for analysis
        Must be enabled by setting BinPlaceILLinkTrimAssembly=true
   -->
   <ItemGroup Condition="'$(BinPlaceILLinkTrimAssembly)' == 'true'">
-    <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/trimmed</RuntimePath>
+    <BinPlaceTargetFrameworks Include="$(BuildSettings)">
+      <RuntimePath>$([MSBuild]::NormalizePath('$(LibrariesTrimmingArtifactsRoot)', 'trimmed'))</RuntimePath>
       <ItemName>TrimmedItem</ItemName>
-    </BinPlaceTargetFramework>
-    <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/reports</RuntimePath>
+    </BinPlaceTargetFrameworks>
+    <BinPlaceTargetFrameworks Include="$(BuildSettings)">
+      <RuntimePath>$([MSBuild]::NormalizePath('$(LibrariesTrimmingArtifactsRoot)', 'reports'))</RuntimePath>
       <ItemName>TrimmingReport</ItemName>
-    </BinPlaceTargetFramework>
-    <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/pretrimmed</RuntimePath>
+    </BinPlaceTargetFrameworks>
+  </ItemGroup>
+
+  <!-- Binplacing for pre-trimming that is used when running the linker
```

Comment:
I'm not sure I understand why we are making these changes. In my mind this shouldn't be changing how we are linking during the library build at all. The library build should stay the same. The only difference is one more step at the end of the build that runs the ILLinker again (on the already trimmed assemblies) but this time with the warnings enabled and the whole shared fx together.

Reply:
Yup, thanks for the clarification on running this new step on the trimmed assemblies, not pre-trimmed. The highlighted changes were to create a directory with pre-trimmed assemblies, which we don't need. Will remove.